manifests/charts/gateways/istio-egress/values.yaml

```diff
@@ -59,6 +59,11 @@ gateways:
 
     configVolumes: []
     additionalContainers: []
+
+    serviceAccount:
+      # Annotations to add to the service account
+      annotations: {}
```

Comment:
This is an array? If yes, should we initialize it to `[]`?

Reply:
Annotations, like labels, are key/value maps

https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#attaching-metadata-to-objects

Reply:
Yes, of course. Not sure why I had list on my mind when reviewed it first time around.